### ISSUE TITLE
Question: getting joints in blueprints

### ISSUE THREAD

**Issue by xmak** [secretlocation/azure-kinect-unreal#14]
Can I use this plugin to get joint positions and orientations in blueprints without applying it to a skeleton?
(I wanto to apply joint positions to some objects in the scene)

**Comment by dylandevs** [comment]
Yeah, for sure! The BP node `Get Joints` can be used to get individual joint positions. The indices in the joint struct match the ones used in `BP_TestActor`.

**Comment by Zyperworld** [comment]
Hi Dylan,

Amazing plugin, started to use it in combination with VR headset & leap motion for better hands tracking.

Is there any way to have the videos stream directly inside unreal ? I had this feature on my Kinect 2 and it was super usefull but I'm not good enough in C++ to develop it by myself for now.

**Comment by MrPantherM** [comment]
Would love to see depth vid and RGB vid in Unreal - similar to Keijiro
 AKVFX for Unity - dying for this actually
